staging/src/k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/fuzzer/fuzzer.go

```diff
@@ -17,6 +17,7 @@ limitations under the License.
 package fuzzer
 
 import (
+	"k8s.io/utils/ptr"
```

Comment:
non-standard imports go in the next block - did you format this by hand or did goimports screw it up?